src/main/java/com/griddynamics/scenario/jagger/JHttpUserScenario.java

```diff
@@ -49,6 +47,10 @@ public String getScenarioName() {
         return scenarioName;
     }
 
+    public List<JHttpUserScenarioStep> getUserScenarioSteps() {
+        return userScenarioSteps;
+    }
+
```

Comment:
in this case we don't need method:  public JHttpUserScenarioStep getUserScenarioStep(int index) {